examples/index.md

```diff
@@ -126,3 +126,38 @@ Content-Type: application/vnd.api+json
 for readability. In practice, these characters must be percent-encoded, as
 noted in the base specification.
 
+## Pagination Links
+
+Example of how to add [pagination links](http://jsonapi.org/format/#fetching-pagination).
+
+Basic request:
+
+```http
+GET /articles?page=3&per_page=1
+```
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/vnd.api+json
+
+{
+  "data": [
+    {
+      "type": "articles",
+      "id": "1",
+      "attributes": {
+        "title": "JSON API paints my bikeshed!",
+        "body": "The shortest article. Ever.",
+        "created": "2015-05-22T14:56:29.000Z",
+        "updated": "2015-05-22T14:56:28.000Z"
+      }
+    }
+  ],
+  "links": {
+    "first": "?page=1&per_page=1",
```

Comment:
I'm pretty sure these need to be complete URIs, not just query strings, since a JSON API document doesn't have a base uri. The spec could specify in the future that the `self` link is the base uri, but right now it doesn't (and the `self` link isn't even mandatory).